amqp/dotnet/env.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "MQ_ENDPOINTS": [
+    {
```

Comment:
strip the env.json to the endpoint that you are expecting to use.

Reply:
Removed the redundant Endpoints.